Source/EasyNetQ.Management.Client/ManagementClient.cs

```diff
@@ -648,6 +648,23 @@ public Task RebalanceQueuesAsync(CancellationToken cancellationToken = default)
         return PostAsync(Rebalance / "queues", cancellationToken);
     }
 
+    public Task CreateShovelAsync(
+        string vHostName,
+        string name,
+        ParameterShovelValue parameters,
+        CancellationToken cancellationToken = default)
+    {
+        if (string.IsNullOrEmpty(vHostName))

```

Comment:
We don't handle the parameter in such a way in other methods, please, remove this if.

Reply:
I've added it because the api wants the http entity `%2F` if virtual host is not specified. I've seen `aliveness-test` adds it automatically, i think because of `VHost` class. Should i use `VHost.Name`?

![image](https://user-images.githubusercontent.com/58050537/220588641-bb1af152-1f5c-42c4-83e9-5df2e27d68c2.png)

Reply:
>I've added it because the api wants the http entity %2F if virtual host is not specified

Yep, you are correct. 

> I've seen aliveness-test adds it automatically

I can't confirm it does, in RabbitMqFixture vhost is provided for IsAliveAsync.

<img width="697" alt="image" src="https://user-images.githubusercontent.com/664889/220594758-66aa66eb-06ff-4f81-8333-39d95533fbbd.png">


<img width="753" alt="image" src="https://user-images.githubusercontent.com/664889/220594636-6e8ba510-f091-4c96-bf64-89b01dc63590.png">

> think because of VHost class. Should i use VHost.Name

In a case of tests, I am ok with it.

At the moment, it seems we don`t have any constant defined for the default virtual host. Additionally, we should consider adding extensions that include a default virtual host parameter to simplify invocations in simple cases. I reckon this could be done separately.

Reply:
Tried the `VHost `class, but if virtual host is not specified, it is not defaulted to `/` (`%2F`).
For the specific shovel route it seems mandatory to me (based on management API documentation), but i'm open to alternative suggestions

Reply:
> At the moment, it seems we don`t have any constant defined for the default virtual host. Additionally, we should consider adding extensions that include a default virtual host parameter to simplify invocations in simple cases. I reckon this could be done separately

Ok, i'll proceed in modifying the tests. And maybe in doing another PR / proposal to manage that default.
Thank you

Reply:
Fixed. Please @Pliner let me know it it's ok for resolution

Reply:
Looks good, thanks.